Docs/eleccion_conf_file.md

```diff
@@ -0,0 +1,32 @@
+# Elección de configuración file
+
+Deno, por defecto, soporta los archivos de configuración con archivos `.json` si los detecta en el directorio de ejecución o superiores. Por tanto, como esto es insuficiente para poder utilizar archivos `.env` y tendremos que crear nuestra propia instancia del mismo. Ahora tenemos 2 opciones, crear el nuestro o encontrar uno de terceros en deno.land o github.
+
+## Criterios de elección
+
+- Tenemos que tener un archivo de configuración que nos permita obtener tanto los .json como los .env
+
+- Que se haya trabajado en este repositorio en los últimos meses, es decir, que esté actualizado
+
+## Opciones
+
+### Creación de nuestro propio archivo de configuración
+
+Esta opcion valida, siempre que lo realicemos así, los criterios. Sin embargo, tiene el problema de que tiene que estar mantenido por nosotros y, si hay algun cambio en los repositorios que se utilicen, entonces fallará.
+
+### (Deno config)[https://deno.land/x/deno_config@v0.1.2]
+
+Está más orientado al mantenimiento a través de "streamline"  la aplicación de deno a traves de cli, .env y archivos json. Sin embargo, la última versión se realizó hace 2 años
```

Comment:
¿Streamline qué viene siendo?

Reply:
A ver, ya he leído el original. No traduzcáis cosas y lo hagáis mal. *streamline* significa hacer una cosa de forma directa. Es cierto que la última versión es de hace dos años, sin embargo.

Reply:
Cierto es, resuelto en [d6b0b75](https://github.com/javigp2002/LazyFood/pull/58/commits/d6b0b75439d79bd3243d4f9d55122bc7b0da2d42)